docs/design/modules/digital_twin_graph/README.md

```diff
@@ -0,0 +1,65 @@
+# Design Specification for Digital Twin Graph Service
+
+- [Introduction](#introduction)
+- [Architecture](#architecture)
+- [Operations](#operations)
+
+## <a name="introduction">Introduction</a>
+
+Ibeji today provides the foundations for constructing and interacting with a digital twin on an edge device. These are low-level abilities and do not necessarily
```

Comment:
I think I'd just remove the word "today". Including that makes me think that this document is a design proposal for a feature that does not yet exist

Reply:
On a similar note, there's a lot of future tense in this document used to describe features that are introduced in this PR. I think it would be more appropriate to use present tense when describing them, since those features are not planned features but rather things that will exist when this document is checked in